controller/user-controller.js

```diff
@@ -0,0 +1,41 @@
+const {userService} = require('../services/user-services');
+
+const signUp = async (req, res) => {
+  try {
+    const {email, password, firstname, lastname, skintype} =req.body;
+
+    await userService.signUp(email, password, firstname, lastname, skintype);
+
+    if (!email|| !password || !firstname || !lastname){
+      throw new Error("KEY Error!");
+    }
+
+    return res.status(201).send({message: "SIGNUP_SUCCESS"});
+  }catch (err) {
+    res.status(err.statusCode || 400).json({message: err.message});
+  }
+};
+
+const signIn = async (req, res) => {
+  try{
+    const {email, password }= req.body;
+
+   const accessToken = await userService.signIn(email,password);
+   
+   
+   if( !eamil || !password ){
+    throw new Error("SIGN IN FAILED!");
+   }
```

Comment:
signIn 함수와 마찬가지로, 위 두가지 코드 순서가 변경되어야 할 것 같습니다.
그리고 오타가 있으니, push 전에 postman을 활용하여 api가 잘 동작하는지 확인해주는 것도 좋으니 참고해주세요~!

```suggestion
   if( !email || !password ){
    throw new Error("SIGN_IN_FAILED!");
   };
   
   const accessToken = await userService.signIn(email, password);
```